src/Cvl/LDAPWrapper/LDAPWrapper.php

```diff
@@ -661,10 +662,17 @@ protected function getAllGroupDNs() {
 
 	/**
 	 *
+	 * @param bool $includeDisabled - Includes disabled users. (can be used only with ActiveDirectory)
 	 * @return array of LDAPUser objects representing all users that are on userBaseDNs path, null if there are no such users.
 	 */
-	public function getAllUsers() {
-		$filter = 'objectClass=user';
+	public function getAllUsers($includeDisabled = true) {
+		$isPerson = 'objectCategory=person';
```

Comment:
Why switch from objectClass to objectCategory? Please check the rest of the code base for remaining uses of objectClass, because the two different conditions could clash now.